zhazam/Modules/Menu/Storages/MainMenuStorage.swift

```diff
@@ -17,7 +17,9 @@ struct MainMenuStorage: CategoryStorageProtocol {
     init() {
         categories = [CategoryViewModel(title: R.string.localizable.play(),
                                         viewController: MenuViewController(storage: GameModesStorage())),
-                      CategoryViewModel(title: R.string.localizable.leaderboard(), viewController: nil),
+                      CategoryViewModel(title: R.string.localizable.leaderboard(),
+                                        viewController: LeaderBoardViewController(
+                                            storage: LeaderBoardStorage())),
```

Comment:
we are now instantiating viewControllers not in a storage. we do it in MainMenuViewController (it is not actually right, but do so for now, I will refactor it in a while)

Reply:
Yes, sir!